articles/tutorials/explore-entanglement.md

```diff
@@ -152,7 +152,7 @@ The arguments to an operation are specified as a tuple, within parentheses.
 The return type of the operation is specified after a colon. In this case, the
 `SetQubitState` operation has no return, so it is marked as returning `Unit`. This is the
 Q# equivalent of `unit` in F#, which is roughly analogous to `void` in C#, and
-an empty tuple (`Tuple[()]`) in Python.
+an empty tuple (`tuple[()]`) in Python.
```

Comment:
The Python value `()` can also be written as `tuple([])`, but not as `tuple[()]`. On the other hand, parameters to type hints in Python are denoted by `[]` rather than `()`, so the type of `tuple([])` is `Tuple[()]`.

Reply:
So I guess we can close this PR, right?

Reply:
I don't think so, it's a genuine confusion we can address! I went with `Tuple[()]` as that's most directly analogous to `unit` in F# and `void` in C#. The Python value `()` or `tuple([])` is most directly analogous to F#'s `()` rather than `unit`. That said, as you point out, type hints aren't something widely known in Python yet, so it may be helpful to take that feedback as impetuous to clarify:

```suggestion
an empty tuple in Python (`()`, represented by the type hint `Tuple[()]`).
```